components/integrations/confirm_integration/confirm_integration.jsx

```diff
@@ -222,6 +224,40 @@ export default class ConfirmIntegration extends React.Component {
                     />
                 </p>
             );
+        } else if (this.state.type === Constants.Integrations.BOT && bot) {
+            const botToken = (new URLSearchParams(this.props.location.search)).get('token');
+
+            headerText = (
+                <FormattedMessage
+                    id='bots.manage.header'
+                    defaultMessage='Bot Accounts'
+                />
+            );
+            helpText = (
+                <p>
+                    <FormattedMarkdownMessage
+                        id='bots.manage.created.text'
+                        defaultMessage='Your Bot Account **{botname}** has been created successfully. Please use the following Bot Access Token to connect to the bot (see documentation for further details).'
```

Comment:
Propose:

```
Your bot account **{botname}** has been created successfully. Please use the following access token to connect to the bot (see documentation for further details).
```

Please make `documentation` a clickable link to https://mattermost.com/pl/default-bot-accounts

Reply:
Note: I haven't added a text about adding the bot to teams and channels.

I'll create a separate ticket to revisit the UX for adding bots to teams/channels. I won't bother making a text change here before the final proposal.

Reply:
@crspeller on above